AlgorithmFactory/Loader.cs

```diff
@@ -184,6 +184,11 @@ private bool TryCreatePythonAlgorithm(string assemblyPath, out IAlgorithm algori
                 return false;
             }
 
+            var pythonnet = GetType().Assembly.GetReferencedAssemblies()
```

Comment:
_referenced_ assemblies aren't always the same as _loaded_ assemblies.

Has this been tested locally?  Also, I think we need to update our pythonnet to include the correct assembly version in `SharedAssemlyInfo.cs`